src/control/server/ioserver/exec_test.go

```diff
@@ -53,6 +53,8 @@ func TestMain(m *testing.M) {
 	case "":
 		os.Exit(m.Run())
 	case "RunnerNormalExit":
+		// remove this once we're running so that it doesn't pollute test results
+		os.Unsetenv("LD_LIBRARY_PATH")
```

Comment:
Curious if something bad can happen with this depending on where dependencies are installed

Reply:
I can answer this one because I made the suggestion to Joel as a workaround for the issue he was hitting (test binary now has a dependency on libfabric, et al). This os.Unsetenv() is only ever called in a subprocess that is launched specifically for this test. So it's completely isolated from anything else.

I'm not really happy with this hack, but I wasn't able to find anything less clunky without major surgery for questionable benefit.

Reply:
Fine to leave it - it's just a bit odd to me that the library path should vary for different tests.